var/spack/repos/builtin/packages/vim/package.py

```diff
@@ -35,8 +35,9 @@ class Vim(AutotoolsPackage):
     """
 
     homepage = "http://www.vim.org"
-    url      = "https://github.com/vim/vim/archive/v8.0.0134.tar.gz"
+    url      = "https://github.com/vim/vim/archive/v8.0.1376.tar.gz"
```

Comment:
Not opposed to this, but wanted to point out that ,in general, there's no need to update `url` at every version change.